ui/app/src/components/AsyncVideoPlayer.tsx

```diff
@@ -17,45 +17,64 @@
 
 import React, { useState, useEffect } from 'react';
 import VideoPlayer from './VideoPlayer';
-
-interface AsyncVideoPlayerProps {
-  // list props...
-  nonPlayableMessage: string;
-}
+import { AsyncVideoPlayerProps } from '../models/AsyncVideoPlayerProps';
+import { PlayerOptions } from '../models/PlayerOptions';
```

Comment:
`PlayerOptions` is nothing more than the `VideoPlayerProps` interface so let's move the interface to the VideoPlayer component file.

If this makes the models directory empty, add a `.gitkeep` file on to it as we will certainly need it down the road for models that are actually shared.

You also don't need to import it. Your IDE will infer it.